dmt/settings_shared.py

```diff
@@ -26,7 +26,6 @@
 ]
 
 TEMPLATE_CONTEXT_PROCESSORS += [  # noqa
-    'django.contrib.messages.context_processors.messages',
```

Comment:
if you're suggesting the messaging stuff shouldn't be in ccnmtlsettings, we can talk about that